proto/tendermint/abci/types.proto

```diff
@@ -6,16 +6,33 @@ option go_package = "github.com/tendermint/tendermint/abci/types";
 // For more information on gogo.proto, see:
```

Comment:
Cannot seem to comment on it directly, but `TxEvent` comment still references `BeginBlock` and `EndBlock`.